Scala/Scala.sublime-syntax

```diff
@@ -1427,32 +1427,32 @@ contexts:
         1: support.function.scala
         2: punctuation.definition.string.begin.scala
       push:
-        - include: f_string
         - meta_include_prototype: false
-        - meta_scope: string.quoted.interpolated.scala
+        - meta_scope: meta.string.interpolated.scala string.quoted.double.block.scala
         - match: '"""'
           scope: punctuation.definition.string.end.scala
           pop: true
+        - include: f_string
     - match: '({{alphaid}})(""")'
       captures:
         1: support.function.scala
         2: punctuation.definition.string.begin.scala
       push:
         - meta_include_prototype: false
-        - meta_scope: string.quoted.triple.interpolated.scala
+        - meta_scope: meta.string.interpolated.scala string.quoted.double.block.scala
```

Comment:
Similar comment as the above.

Reply:
The distinction is `.block` subscope from Python.